ElectronicObserver/FormMainWpf.xaml

```diff
@@ -357,6 +357,16 @@
 						<Image Source="{Binding ArsenalImageSource}" />
 					</MenuItem.Icon>
 				</MenuItem>
+				<MenuItem
+					d:Header="Arsenal"
```

Comment:
Either add the English name here or remove this. `d:` is design time only, I'm not even sure why I kept those in here, actually.